sphinx/changelog.rst

```diff
@@ -0,0 +1,602 @@
+Changelog
```

Comment:
This file results from a failed documentation build, could you remove it?
If you want to, you can also directly add it to `.gitignore` to prevent similar problems in the future.